### PR TITLE
AW-81 - Maps Problem

### PR DESCRIPTION
**Pull request by spg**
https://arcbees.atlassian.net/browse/AW-81

### CODE REVIEW

**Comment by meriouma** [comment]
LGTM

File: src/main/java/com/arcbees/website/client/application/maps/GwtMapsLoader.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.arcbees.website.client.application.maps;
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/website/client/application/maps/GwtMapsLoader.java]: missing Arcbees license header

File: src/main/java/com/arcbees/website/client/application/maps/GwtMapsLoaderImpl.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package com.arcbees.website.client.application.maps;
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/website/client/application/maps/GwtMapsLoaderImpl.java]: missing Arcbees license header

File: src/main/java/com/arcbees/website/client/application/maps/MapsModule.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.arcbees.website.client.application.maps;
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/website/client/application/maps/MapsModule.java]: missing Arcbees license header

**Comment by jasonlemay** [comment]
LGTM (after licenses)

**Pull request merged**